lua/telescope/actions/init.lua

```diff
@@ -59,7 +59,7 @@ function actions.get_current_line()
   return action_state.get_current_line()
 end
 
-function actions.get_current_picker(prompt_bufnr)
+function action_state.get_current_picker(prompt_bufnr)
```

Comment:
This needs to be `actions`.

Otherwise thanks :)

Reply:
Thanks @Conni2461 I think this is resolved now :-)